magicctapipe/scripts/lst1_magic/lst_m1_m2_cloud_correction.py

```diff
@@ -379,11 +384,32 @@ def main():
 
     df_all = df_all.drop(columns=["deviation"], errors="ignore")
 
+    Path(args.output_dir).mkdir(exist_ok=True, parents=True)
+    input_file_name = Path(args.input_file).name
+    output_file_name = input_file_name.replace("dl1_stereo", "dl1_corr")
+    output_file = f"{args.output_dir}/{output_file_name}"
+
     save_pandas_data_in_table(
-        df_all, args.output_file, group_name="/events", table_name="parameters"
+        df_all, output_file, group_name="/events", table_name="parameters"
+    )
+
+    subarray_info.to_hdf(output_file)
+
+    correction_params = config.get("cloud_correction", {})
+
+    log_dir = args.output_dir
+    log_filename = "cloud_correction.log"
+    log_filepath = os.path.join(log_dir, log_filename)
+
+    logging.basicConfig(
+        filename=log_filepath,
+        level=logging.INFO,
+        format="%(asctime)s - %(levelname)s - %(message)s",
     )
 
-    subarray_info.to_hdf(args.output_file)
+    logging.info(f"Correction parameters: {correction_params}")
+    logging.info(f"ctapipe version: {ctapipe.__version__}")
+    logging.info(f"magicctapipe version: {magicctapipe.__version__}")
```

Comment:
why do you use a separate log? you have "logger" defined before and used just below this line. you can use it also to write the correction parameters and version of software